docs/pages/accounts/account-types.md

```diff
@@ -2,69 +2,115 @@
 title: Account Types
 ---
 
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+There are three types of Expo accounts; Personal, Team, and Organization. The type you choose depends on whether you work alone on projects or in a group. However, you must start with at least one Personal Account to use Expo.
```

Comment:
I advised that we still include Team Accounts in the documentation for now since [this discussion](https://exponent-internal.slack.com/archives/CTR08334G/p1659653015542189) is still pending

Reply:
We'll have to conclude internally as to whether include Team related info in docs or not. As Kelley mentioned, I followed her suggestion.

My suggestion, is to keep the info for now since its still a feature available but remove it as soon as Team accounts functionality are disabled.